torch/distributed/_spmd/distribute.py

```diff
@@ -365,6 +365,8 @@ def default_factory_op_rule(
 # without DTensor inputs.
 FACTORY_OPS: Dict[torch._ops.OpOverload, Callable] = {
     aten.scalar_tensor.default: default_factory_op_rule,
+    aten.arange.start: default_factory_op_rule,
```

Comment:
this looks OK to me. We might need to consolidate this one and the one in `FACTORY_SYM_INT_CONSUMERS` in follow up PRs.